packages/constants/assets/local-asset-registry.json

```diff
@@ -172,4 +172,4 @@
       }
     ]
   }
-]
\ No newline at end of file
+]
```

Comment:
Linting complained about the lack of trailing newline — not sure how this got committed to `main`